produce bit-identical results
by not embedding timestamps in gz files

Index: kakoune-0.0+git.20170223/src/Makefile
===================================================================
--- kakoune-0.0+git.20170223.orig/src/Makefile
+++ kakoune-0.0+git.20170223/src/Makefile
@@ -73,7 +73,7 @@ kak : $(objects)
 # Generate the man page
 ../doc/kak.1.gz: ../doc/kak.1.txt
 	a2x --no-xmllint -f manpage $<
-	gzip -f $(basename $<)
+	gzip -n -9 -f $(basename $<)
 
 # Generate the editor's documentation pages
 # Since `a2x` won't generate man pages if some sections are missing (which we don't need),
@@ -82,7 +82,7 @@ kak : $(objects)
 	a2x --no-xmllint -f manpage $<
 	sed -i -r -e "s,^\.TH .+,.TH KAKOUNE 1 \"\" \"\" \"$(basename $(notdir $<))\"," \
 	-e "/^\.SH \"NAME\"/{N;d;}" $(@:.gz=.1)
-	gzip -f $(@:.gz=.1)
+	gzip -n -9 -f $(@:.gz=.1)
 	mv -f $(@:.gz=.1.gz) $@
 
 check: test
